packages/components/src/LightBox/LightBox.test.tsx

```diff
@@ -1,135 +1,192 @@
 import React from "react";
-import { fireEvent, render } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import {
+  BREAKPOINT_SIZES,
+  mockViewportWidth,
+} from "@jobber/hooks/useBreakpoints";
 import { LightBox } from ".";
 
-test("Lightbox opens and shows the image", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
-  const handleClose = jest.fn();
-
-  const { queryByText } = render(
-    <LightBox
-      open={true}
-      images={[
-        {
-          title: title,
-          caption: caption,
-          url: "https://i.imgur.com/6Jcfgnp.jpg",
-        },
-      ]}
-      onRequestClose={handleClose}
-    />,
-  );
-
-  expect(queryByText(title)).toBeInstanceOf(HTMLElement);
-  expect(queryByText(caption)).toBeInstanceOf(HTMLElement);
-});
-
-test("lightbox doesn't show up", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
-  const handleClose = jest.fn();
-
-  const { queryByText } = render(
-    <LightBox
-      open={false}
-      images={[
-        {
-          title: title,
-          caption: caption,
-          url: "https://i.imgur.com/6Jcfgnp.jpg",
-        },
-      ]}
-      onRequestClose={handleClose}
-    />,
-  );
-
-  expect(queryByText(title)).toBeNull();
-  expect(queryByText(caption)).toBeNull();
-});
+const { setViewportWidth } = mockViewportWidth();
 
-test("lightbox closes when user clicks close", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
+describe("LightBox", () => {
+  beforeEach(() => {
+    setViewportWidth(BREAKPOINT_SIZES.lg);
+  });
+  test("Lightbox opens and shows the image", () => {
+    const title = "Dis be a title";
+    const caption = "Dis be a caption 🎉";
+    const handleClose = jest.fn();
+
+    const { queryByText } = render(
+      <LightBox
+        open={true}
+        images={[
+          {
+            title: title,
+            caption: caption,
+            url: "https://i.imgur.com/6Jcfgnp.jpg",
+          },
+        ]}
+        onRequestClose={handleClose}
+      />,
+    );
 
-  //This expect is called from the close function as the react-image-lightbox
-  //library has an easing animation that occurs before the close function is
-  //called. This avoids an arbitrary setTimeout in our test.
-  const handleClose = jest.fn(() => {
-    expect(handleClose).toHaveBeenCalledTimes(1);
+    expect(queryByText(title)).toBeInstanceOf(HTMLElement);
+    expect(queryByText(caption)).toBeInstanceOf(HTMLElement);
   });
 
-  const { getByLabelText } = render(
-    <LightBox
-      open={true}
-      images={[
-        {
-          title: title,
-          caption: caption,
-          url: "https://i.imgur.com/6Jcfgnp.jpg",
-        },
-      ]}
-      onRequestClose={handleClose}
-    />,
-  );
-
-  fireEvent.click(getByLabelText("Close"));
-});
+  test("lightbox doesn't show up", () => {
```

Comment:
I know this was an existing test assertion but I don't really get _why_ the lightbox wouldn't show up in this scenario.

you have to examine the props to understand what's different about this setup and leads to the result which I think could be improved with a `context` wrapper

since it was already like this, it's probably fine to leave it as is, but if you're up for a small refactor of the tests I think it would really help make this test suite easier to understand!